sirepo/package_data/static/js/sirepo-plotting.js

```diff
@@ -437,13 +437,7 @@ SIREPO.app.factory('plotting', function(appState, frameCache, panelState, utilit
 
     function initPlot(scope) {
         var interval = null;
-        var requestData = function(forceRunCount) {
```

Comment:
Looks fine to me. I don't remember why we had to do this...

Reply:
I think it was a work-around for apps which were trying to plot partially-created datafiles. We've gotten better at handling this on the server, so the client doesn't need the retry feature.